content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -146,8 +146,8 @@ Refer to [Logging Architecture](/docs/concepts/cluster-administration/logging/)
 
 ## Object Count Quota
 
-You can set quota for the total number of certain resources of all standard,
-namespaced resource types using the following syntax:
+You can set quota for **the total number of one particular resource** of all standard, namespaced resource types 
+using the following syntax:
```

Comment:
My experiments showed that the `<resource>` in `count/<resource>` can be anything.
It can be something that doesn't exist at all.

Reply:
So ... this doen't have to be a "standard" resource type, right?

Reply:
Although the API accepts any resource name in that field, this could mean:
1. you can set any ResourceQuota, even for an API the cluster doesn't yet have
   - if the cluster gains that resource kind, the quota starts to be enforced
1.  you can set any ResourceQuota, even for an API the cluster doesn't yet have
    - however, there is an internal list of API kinds where enforcement happens
    - if you set a ResourceQuota for an different resource kind, the ResourceQuota is not enforced, and the lack of enforcement is silent

Let's be clear on which of those two cases applies.

Reply:
I think the 1st case is the correct answer, but has to double confirm this.

Reply:
It's case 2 (very well considered).  Though the "default" aggregated apiserver (APIService) enforces quota.

Reply:
'standard' is indeed ambiguous, better to clearly list out: built-in API and CRD